tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
@TeslaDethray would you go further to say that the Behat would cover the public api "happy path" and we would focus the Command units on the edges? In some ways I think of gherkin as a view specs where I am not sure it's the right place to test input validation and exceptions for instance.

Do you expect failure scenarios to be described in gherkin?